odoo/addons/base/views/res_users_views.xml

```diff
@@ -274,28 +274,30 @@
                     <field name="active"/>
                     <field name="login_date"/>
                     <templates>
-                        <t t-name="kanban-box">
-                            <div t-attf-class="oe_kanban_global_click">
-                                <div class="o_kanban_image">
-                                    <img alt="Avatar" t-att-src="kanban_image('res.users', 'avatar_128', record.id.raw_value)"/>
-                                </div>
-                                <div class="oe_kanban_details">
-                                    <ul>
-                                        <li class="text-success float-end mb4" t-if="record.active.raw_value"><i class="fa fa-circle" role="img" aria-label="Ok" title="Ok"></i></li>
-                                        <li class="text-danger float-end mb4" t-if="!record.active.raw_value"><i class="fa fa-circle" role="img" aria-label="Invalid" title="Invalid"></i></li>
-                                        <li class="mb4">
-                                            <strong><field name="name"/></strong>
-                                        </li>
-                                        <li class="d-flex flex-wrap">
-                                            <span class="mb4 text-truncate" title="Login">
-                                                <i class="fa fa-envelope me-1" role="img" aria-label="Login"/>
-                                                <field name="login"/>
-                                            </span>
-                                            <field class="badge rounded-pill mb4 ms-auto" name="lang"/>
-                                        </li>
-                                    </ul>
+                        <t t-name="kanban-card" class="row g-0">
+                            <aside class="col-2">
+                                <field name="avatar_128" widget="image" alt="Avatar"/>
+                            </aside>
+                            <main class="col-10 ps-2">
+                                <div>
+                                    <div class="text-success float-end mb4" t-if="record.active.raw_value">
+                                        <i class="fa fa-circle" role="img" aria-label="Ok" title="Ok"></i>
+                                    </div>
+                                    <div class="text-danger float-end mb4" t-if="!record.active.raw_value">
+                                        <i class="fa fa-circle" role="img" aria-label="Invalid" title="Invalid"></i>
+                                    </div>
+                                    <div class="mb4">
+                                        <field class="fw-bold" name="name"/>
+                                    </div>
```

Comment:
```suggestion
                                    <field class="fw-bold mb4" name="name"/>
```
This is a general comment: stop wrapping fields in div/span, it's not necessary (most of the time)